Updates to wK Docs (#5995)
scalableminds/webknossos: docs/datasets.md

### Uploading through the File System (Self-Hosted Instances Only)
On self-hosted instances, large datasets can be efficiently imported by placing them directly in the file system:

* Place the dataset at `<webKnossos directory>/binaryData/<Organization name>/<Dataset name>`. For example `/opt/webknossos/binaryData/Springfield_University/great_dataset`.
* Go to the [dataset view on the dashboard](./dashboard.md)
* Use the refresh button on the dashboard or wait for webKnossos to detect the dataset (up to 10min)

Typically webKnossos can infer all the required metadata for a dataset automatically and import datasets automatically on refresh. In some cases, you will need to manually import a dataset and provide more information:
* On the dashboard, click *Import* for your new dataset
* Provided the requested properties, such as *scale* and *largestSegmentId*. See the section on [configuring datasets](.datasets.md#configuring-datasets) below for more detailed explanations of these parameters.

> !!! info

Author: yes

...
```

### Converting Datasets
Any dataset uploaded through the web interface at [webknossos.org](https://webknossos.org) is automatically converted for compatibility.

For manual conversion, we provide the following software tools and libraries:
- The [webKnossos Cuber](https://docs.webknossos.org/wkcuber/index.html) is a CLI tool that can convert many formats to WKW. 
- For other file formats, the [Python webKnossos libray](https://docs.webknossos.org/webknossos-py/index.html) can be an option for custom scripting.

> See page on [software tooling](.tooling.md) for more.

Reviewer: ```suggestion
See page on [software tooling](./tooling.md) for more.
```

To make changes, click on the "Settings" action next to a dataset in the "My Datasets" tab of your dashboard.
Editing these settings requires your account to have enough access rights and permissions. [Read more about this.](./users.md)

### Data Tab
The *Data* tab contains the settings for correctly reading the dataset as the correct data type (e.g., `uint8`), setting up, and configuring any layers.

- `Scale`: The physical size of a voxel in nanometers, e.g., `11, 11, 24`

For each detected layer:
- `Bounding Box`: The position and extents of the dataset layer in voxel coordinates. The format is `x, y, z, x_size,y_size, z_size` or respectively `min_x, min_y, min_z, (max_x - min_x), (max_y - min_y), (max_z - min_z)`.
- `Largest Segment ID`: The highest ID that is currently used in the respective segmentation layer. This is required for volume annotations where new objects with incrementing IDs are created. Only applies to segmentation layers.

> The `Advanced` view lets you edit the underlying [JSON configuration](./data_formats.md#wkw-metadata-by-example) directly. Toggle between the `Advanced` and `Simple` page in the upper right. Advanced mode is only recommended for

Reviewer: the sentence's end is missing, I guess.

> webKnossos automatically periodically checks and detects changes to a dataset's metadata (`datasource-properties.json`) on disc. (Only relevant for self-hosted instances) Before applying these suggestions, users can preview all the new settings (as JSON) and inspect just the detected difference (as JSON).

Reviewer: I think, we typically write american english.

```suggestion
webKnossos automatically periodically checks and detects changes to a dataset's metadata (`datasource-properties.json`) on disk (only relevant for self-hosted instances). Before applying these suggestions, users can preview all the new settings (as JSON) and inspect just the detected difference (as JSON).
```

- `Teams allowed to access this dataset`: Defines which [teams of your organization](./users.md) have permission to work with this dataset. By default, no team has access, but users with *admin* and *team manager* roles can see and edit the dataset.
- `Sharing Link`: A web URL pointing to this dataset for easy sharing that allows any user to view your dataset. The URL contains an access token to allow people to view the dataset without a webKnossos account. The access token is random, and therefore the URL cannot be guessed by visitors. You may also revoke the access token to create a new one. Anyone with a URL containing a revoked token will no longer have access to this dataset. 
Read more in [the Sharing guide](./sharing.md).

// Todo add image

### Metadata Tab

> - `Display Name`: A meaningful name for a dataset other than its automatically assigned technical moniker/ID. Displayed in various parts of webKnossos. Especially useful when sharing datasets with outsiders, when you want to "hide" any internal nameing schemes or make it more approachable, e.g., `L. Simpson et al.: Full Neuron Segmentation` instead of `neuron_seg_v4_2022`.

Reviewer: ```suggestion
- `Display Name`: A meaningful name for a dataset other than its (automatically assigned) technical name which is usually limited by naming rules of file systems. It is displayed in various parts of webKnossos. The display name may contain special characters and can also be changed without invalidating already created sharing URLs. It can also be useful when sharing datasets with outsiders while "hiding" any internal naming schemes or make it more approachable, e.g., `L. Simpson et al.: Full Neuron Segmentation` instead of `neuron_seg_v4_2022`.
```

// Todo add image

### View Configuration Tab
The *View configuration* tab lets you set defaults for viewing this dataset. Anytime a user opens a dataset or creates a new annotation based on this dataset, these default values will be applied. 

Defaults include:
- `Position`: Default position of the dataset in voxel coordinates. When opening the dataset, users will be located at this position.
- `Zoom`: Default zoom.
- `Interpolation`: Whether interpolation should be enabled by default.
- `Layer Configuration`: Advanced feature to control the default settings on a per-layer basis. It needs to be configured in JSON format. E.g., layer visibility & opacity, color, contrast/brightness/intensity range ("histogram sliders"), and many more.

![Dataset Editing: View Configuration Tab](images/dataset_view_config.png)

Of course, the defaults can all be overwritten and adjusted once a user opens the dataset in the main webKnossos interface and makes changes to any of these settings in his viewports. 

For self-hosted webKnossos instances, there are two ways to set default *View Configuration* settings:
- in the web UI as described above

> - inside the `datasource_properties.json` on disc

Reviewer: ```suggestion
- inside the `datasource_properties.json` on disk
```

This may be useful when dealing with large datasets that should remain in your data center.
[Please contact us](mailto:hello@webknossos.org) or [write a post](https://forum.image.sc/tag/webknossos), if you require any assistance with your setup.

scalable minds also offers a dataset alignment tool called *Voxelytics Align*.
[Learn more.](https://scalableminds.com/voxelytics-align)

![Dataset Alignment](https://www.youtube.com/watch?v=yYauIHZcI_4)

## Sample Datasets

> For convience and testing, we provide a list of sample datasets for webKnossos:

Reviewer: ```suggestion
For convenience and testing, we provide a list of sample datasets for webKnossos:
```
